resources/replace_or_add.rb

```diff
@@ -53,19 +53,7 @@
     temp_file.close
     temp_file.unlink
   end
-else
-  unless new_resource.replace_only
-    converge_by "Updating file #{new_resource.path}" do
-      begin
-        nf = ::File.open(new_resource.path, 'w')
-        nf.puts new_resource.line
-      rescue ENOENT
-        Chef::Log.info('ERROR: Containing directory does not exist for #{nf.class}')
-      ensure
-        nf.close
-      end
-    end
-  end
+
```

Comment:
Creating the new file was a deliberate choice.  We are doing an add.

Reply:
Noted reverting that change 👍